app_dart/lib/src/request_handlers/refresh_github_commits.dart

```diff
@@ -61,8 +61,21 @@ class RefreshGithubCommits extends ApiRequestHandler<Body> {
         config, branchHttpClientProvider, log, gitHubBackoffCalculator);
 
     for (Branch branch in branches) {
+      int hours = config.existingBranchHours;
+
+      if (branch.name != 'master') {
+        final Set<Commit> commits = await datastore
+            .queryRecentCommits(limit: 1, branch: branch.name)
```

Comment:
Any reason to limit it to 1? this can potentially limit processing one commit per cron job iteration.

Reply:
Please refer to above comment. The case when limiting 1 happens only when a new release branch is detected for the first time.